apps/files/src/default.js

```diff
@@ -22,6 +22,7 @@ const appInfo = {
   icon: 'folder',
   isFileEditor: false,
   extensions: [],
+  fileActions: window.phoenixConfig.files.actions,
```

Comment:
accessing the window object here feels a bit hacky.

it should be possible to inject the config by adjusting this file to be a big `function(phoenixConfig)` that returns the objects that we currently return in the default export.

the caller should then detect if we returned a function, if yes call it with injection. else assume it's an object

(and I wonder if this will stay relevant once we switch to a separate phoenix-files app)

Reply:
> (and I wonder if this will stay relevant once we switch to a separate phoenix-files app)

I dont think so .... initialization of code like this will happen in a vue component and the config is living in the store then.

So from my pov this is an acceptable work around for the time being.